checks/index.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Alerts & Check-Up'
+backlink_href: /docs
+backlink_title: 'Documentation'
+---
+
+import * as checkUtil from '../util/checks'
+
+export const CheckLinks = () => {
+  const checkTitles = checkUtil.CHECK_TITLES;
+  return (
+    <ul>
+      {Object.entries(checkTitles).map(([checkGroup, items]) => {
+        return (
+          <li key={checkGroup}>
+            <strong>{checkGroup}</strong>
+            <ul>
+              {Object.entries(items).map(([checkName, checkTitle]) => {
+                return (
+                  <li key={checkName}>
+                    <Link to={`/docs/checks/${checkGroup}/${checkName}`}>
+                      {checkTitle}
+                    </Link>
+                  </li>
+                )
+              })}
+            </ul>
+          </li>
+        )
+      })}
+    </ul>
+  )
+}
+
+To help you discover potential problems, pganalyze automatically performs a number
+of checks on your database.
```

Comment:
I wonder if we should include a screenshot here, similar to how we do on the Log Insights and automatic EXPLAIN pages. For example this one:

<img width="1001" alt="Screen Shot 2021-03-31 at 12 51 16 PM" src="https://user-images.githubusercontent.com/7227/114796377-a450e000-9d45-11eb-996e-194d93f0d8a7.png">

Reply:
Yeah, good idea. I'll just use this screenshot.